File: Assignment_35/Program3.java
import java.util.*;

class StringX
{
    public static int Difference(String str)
    {
        int icnt1 = 0, icnt2 = 0;
        int i = 0;
        for(i = 0; i<str.length(); i++)
        {
            char ch = str.charAt(i);

            if(ch >= 'a' && ch  <= 'z')
            {
                icnt1++;
            }
            else if(ch >= 'A' && ch  <= 'Z')
            {
                icnt2++;
            }
        }
        return icnt1 - icnt2;
    }
}
class Program3
{
    public static void main(String[] args) 
    {
        int iRet = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter String :");
        String input = sobj.nextLine();

        StringX xobj = new StringX();

        iRet = xobj.Difference(input);

        System.out.println("Count is :"+iRet);

         
    }
}

File: Assignment_35/Program3.c
#include<stdio.h>

int Difference(char *str)
{
    int icnt1 = 0;
    int icnt2 = 0;
    while(*str != '\0')
    {
        if(*str >= 'a' && *str  <= 'z')
        {
            icnt1++; 
        }
        else if(*str >= 'A' && *str  <= 'Z')
        {
            icnt2++; 
            
        }
        str++;
    }
    return icnt1 - icnt2 ;
}

int main()
{
    char Arr[20];
    int iRet = 0;

    printf("Enter String :\n");
    scanf("%[^'\n']s",Arr);

    iRet = Difference(Arr);

    printf("%d",iRet);

    return 0;
}